### DISCUSSION TITLE
Make resolution selector in options into a dropdown menu

### DISCUSSION THREAD

**Discussion by cooljeanius** [SuperTux/supertux#2094] [Ideas]
**SuperTux version:** 0.6.3
**System information:** macOS Big Sur

##### Expected behavior
Clicking on the "Window Resolution" item in the "Options" menu displays a dropdown menu from which I can select my display's resolution (3072x1920)

##### Actual behavior
Clicking on the "Window Resolution" item in the "Options" menu simply switches to the next resolution available, making it hard to figure out how many more clicks it'll take to get to my desired resolution

##### Steps to reproduce actual behavior
1. Open SuperTux
2. Open the "Options" menu
3. Click the "Window Resolution" item

**Comment by Semphriss** [comment]
I support, there is no need to scroll through all possible resolutions.

I recommend using the recently added string selection menu.
